python/tvm/tir/schedule/schedule.py

```diff
@@ -71,6 +81,8 @@ def __init__(
         debug_mode : Union[bool, int]
             Do extra correctness checking after the class creation and each time
             scheduling primitive
+        error_render_level : str = "detail"
+            The level of error rendering. Choices: "detail", "fast", "none"
```

Comment:
Can you explain the differences in the levels?

Reply:
Yeah sure. I am going to add more docs for those 3 options

Reply:
What about this one:

```
The level of error rendering. Choices: "detail", "fast", "none".
"detail": Render a detailed error message, with the TIR and error locations printed
"fast: Show a simple error message without rendering or string manipulation
"none": Do not show any error message.
```

Reply:
Looks good.